lib/main.dart

```diff
@@ -1,115 +1,31 @@
+// Absolute pathing is used throughout this project
```

Comment:
We can move this to the ContributingMD instead so we can delete it here & I can make the ticket for it